packages/client/src/components/dialog/dialog.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import {
+  Dialog as MuiDialog,
+  DialogProps,
+} from '@material-ui/core';
+
+/**
+ * Wrapper for Material UI Dialog class to allow us to set default behavior for
+ * onClose().
+ */
+export const Dialog = (props: DialogProps) => {
+  const { onClose, children, ...otherProps } = props;
+
+  const handleClose = (event: {}, reason: 'backdropClick' | 'escapeKeyDown') => {
+    if (reason !== 'backdropClick') {
```

Comment:
Does this break functionality/accessibility for escapeKeyPressing?

Reply:
Nope, using escape still works with this in place.